From: Sven Roederer <SvenRoederer@users.noreply.github.com>
Date: Fri, 12 Oct 2018 16:54:28 +0200
Subject: wizard: do not commit setting when devmode=1

This helps to debug the wizard, by having the changes for review in
the buffer.

diff --git a/utils/luci-app-ffwizard-berlin/luasrc/controller/assistent/assistent.lua b/utils/luci-app-ffwizard-berlin/luasrc/controller/assistent/assistent.lua
index 18f93c504f1c71665bd57c1737d59784043d1344..efe4a42c5ed6e0b07afee002a5f6a65e08156c6e 100644
--- a/utils/luci-app-ffwizard-berlin/luasrc/controller/assistent/assistent.lua
+++ b/utils/luci-app-ffwizard-berlin/luasrc/controller/assistent/assistent.lua
@@ -98,16 +98,20 @@ function commit()
 
   tools.configureQOS()
 
-  uci:commit("dhcp")
-  uci:commit("olsrd")
-  uci:commit("olsrd6")
-  uci:commit("firewall")
-  uci:commit("system")
-  uci:commit("ffwizard")
-  uci:commit("freifunk")
-  uci:commit("wireless")
-  uci:commit("network")
-  uci:commit("qos")
+  if uci:get("system", "ffberlin", "devmode") == "1" then
+    fftools.logger("devmode is enabed; changes will not commited")
+  else
+    uci:commit("dhcp")
+    uci:commit("olsrd")
+    uci:commit("olsrd6")
+    uci:commit("firewall")
+    uci:commit("system")
+    uci:commit("ffwizard")
+    uci:commit("freifunk")
+    uci:commit("wireless")
+    uci:commit("network")
+    uci:commit("qos")
+  end
 
   sys.init.enable("olsrd")
   sys.init.enable("olsrd6")
